design-layout/css-layout-position-assignmnet-3/dashboard-design-layout-assignment-3.1/dashbordLayout.css

```diff
@@ -0,0 +1,247 @@
+body {
```

Comment:
No need to use of vh and vw for every height and width and alwasy try to write common css